pipelines/tekton/aiedge-e2e/aiedge-e2e.bike-rentals.pipelinerun.yaml

```diff
@@ -3,7 +3,9 @@ kind: PipelineRun
 metadata:
   labels:
     tekton.dev/pipeline: aiedge-e2e
-  generateName: aiedge-e2e-
+    model-name: bike-rentals-auto-ml
+
```

Comment:
```suggestion
```

Reply:
The recommendation for model-version was a suggestion IF tekton supported the ability to dynamically add parameters as label `key=value` pairs.  It doesn't look like that exists so we can only provide an example with static values until we have automation that supports creating a pipeline run with the model version as a label value